airbyte-container-orchestrator/src/main/resources/application.yml

```diff
@@ -25,6 +25,9 @@ airbyte:
         name: ${AIRBYTE_API_AUTH_HEADER_NAME:}
         value: ${AIRBYTE_API_AUTH_HEADER_VALUE:}
       host: ${INTERNAL_API_HOST}
+  worker:
+    replication:
+      persistence-flush-period: ${REPLICATION_FLUSH_PERIOD:60}
```

Comment:
Nit:  consider adding the unit type to the config name/env var (e.g. `-sec`, `_SECONDS`).

Reply:
done